QSB/AnglerFish/Patches/AnglerPatches.cs

```diff
@@ -0,0 +1,9 @@
+﻿using HarmonyLib;
+using QSB.Patches;
+
+namespace QSB.AnglerFish.Patches {
+    [HarmonyPatch]
+    internal class AnglerPatches : QSBPatch {
+        public override QSBPatchTypes Type => QSBPatchTypes.OnClientConnect;
+    }
+}
```

Comment:
?

Reply:
just in case i wanted to add patches, which i am now doing